### PR TITLE
NOSTORY: Added readmes for ec2 module examples

### PR DESCRIPTION
**Pull request by patrickmukumbu**
Added some READMEs for the ec2 module examples

### CODE REVIEW

File: examples/complete/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+|------|---------|
+| <a name="provider_aws"></a> [aws](#provider\_aws) | n/a |
+| <a name="provider_null"></a> [null](#provider\_null) | n/a |
+| <a name="provider_tls"></a> [tls](#provider\_tls) | n/a |
+
+## Modules
+
+| Name | Source | Version |
+|------|--------|---------|
+| <a name="module_ec2_instance_complete"></a> [ec2\_instance\_complete](#module\_ec2\_instance\_complete) | ./../.. | n/a |
```
Comment (JohnNMwangi) [inline examples/complete/README.md:17]: In the examples change source to boldlink/ec2/aws and add the version.

**Review by JohnNMwangi — APPROVED** [review]

File: examples/complete/README.md (hunk 1 above)
Comment (JohnNMwangi) [inline examples/complete/README.md:17]: n the examples change source to `boldlink/ec2/aws` and add the version.

**Review by JohnNMwangi — CHANGES_REQUESTED** [review]

File: examples/complete/README.md (hunk 1 above)
Comment (patrickmukumbu) [reply examples/complete/README.md:17]: Alright

**Review by JohnNMwangi — APPROVED** [review]
All good to me !

**Pull request merged**